src/main/java/emissary/directory/DirectoryPlace.java

```diff
@@ -1186,10 +1205,6 @@ public int removePlaces(final List<String> keys) {
      */
     @Override
     public int irdRemovePlaces(@Nullable final List<String> keys, final boolean propagating) {
-        if (this.emissaryNode.isStandalone()) {
```

Comment:
This could have some integration implications...in standalone mode, if a place fails to start the keys should be removed but they're not.

Reply:
If we end up spending too much time on this review we can break this out and get it integrated ahead of time.